hours/tests/test_translation_serializer.py

```diff
@@ -1,11 +1,18 @@
+import datetime
+
 import pytest
+from django.conf import settings
+from pytz import timezone
 
 from hours.models import Resource
 from hours.serializers import ResourceSerializer
 
 
 @pytest.mark.django_db
 def test_to_representation(resource):
+    now = datetime.datetime.now(timezone(settings.TIME_ZONE))
```

Comment:
Could this be `django.utils.timezone.now()`?

Reply:
Good point, also changed the serializer to return UTC times.